py/desidatamodel/test/test_check.py

```diff
@@ -64,7 +64,7 @@ def test_collect_files(self):
         files_to_regexp(self.data_dir, files)
 
         self.assertInLog(log, ("{0}/doc/examples/badModel.rst has no file " +
-                                "regexp!").format(os.environ[DM]))
+                               "regexp!").format(os.environ[DM]))
```

Comment:
Similar comment: the changes are just changes in spacing.

Reply:
Similarly: This change fixed the [Failing unit tests for code style](https://travis-ci.org/github/desihub/desidatamodel/jobs/691486131).